### ISSUE TITLE
Newest version 1.6.10 causes immediate crash

### ISSUE THREAD

**Issue by morthein** [DazedNConfused-/CDDA-Game-Launcher#89]
### Description

Updated to new version as of some 13 hours ago of this message, and it causes the game launcher to crash without opening anything else.

### To Reproduce

1. Start launcher.

### Expected behaviour

Normal startup and then proceeding to the UI window showing the most recent updates.

### Attempted Fixes

Ran in admin
restarted PC after updating
ran CDDA.exe separately and it's not the game, just the launcher/updater itself.

### System Details

```shell
windows 11 - latest build
CDDA version up to day as of 5 days ago.
```

### Additional context

Previous to this update, things worked flawlessly.
Note: I redownloaded 1.6.9 and reinstalled and the launcher works perfectly. Something in 1.6.10 my system doesn't like or is botched. 
It never gets past running the batch file (no UI) but closes in less than a second after attempting to open.

**Comment by Leonlit** [comment]
I tried downloading the new installer in a new Windows Virtual Machine and no crash occurred. You might want to include your log file content here. Try locating the log file at "C:\Users\<username>\AppData\Local\CDDA Game Launcher\app.txt".

**Comment by morthein** [comment]
It doesn't show that I updated it to 1.6.10 since it crashes immediately, but it does show recently that I installed 1.6.9 and ran it (around noon EST) as shown in the last entry.  I even tried reinstalling the 1.6.10 and rerunning it and it just opens and immediately closes without logging in the 'app' file.
________________________________

**Comment by DazedNConfused-** [comment]
Well that's certainly odd. Using a Windows 11 VM I was able to run the launcher with no hitches. Is anyone else experiencing this issue? Any other details that could be provided in order to diagnose the issue?

**Comment by loeuekken** [comment]
Had the same problem. On Win10, German localization, updated to 1.6.10 today and the launcher did not start anymore.

This was the error I got for a fraction of a second:
![grafik](https://user-images.githubusercontent.com/34602382/225127996-7518b08e-4047-4365-8719-a376e81cd247.png)
And this was my solution:
![grafik](https://user-images.githubusercontent.com/34602382/225127894-efcd9a07-4cca-49dd-b935-36d2922900a0.png)
Worked then

**Comment by morthein** [comment]
Is there a new download of your fix?  the default 'update to 1.6.10' prompted from the launcher still does the same crash.

**Comment by morthein** [comment]
I just made the adjustments and it works.   :)   yay thank you.

**Comment by DazedNConfused-** [comment]
Another day, another timezone bug. Will put this in the backlog

**Comment by TadhgEomonn** [comment]
Had same problem. loeuekken's solution worked for me also.

Thanks.

**Comment by Stadler76** [comment]
Took me like two hours of installing and uninstalling every permutation of python 3.9 and 3.10 in both 32bit and 64bit versions because I got a lot of could not load xyz32.dll / could not load built in python module blah-yadda in xyz.py till I gave up and downgraded to 1.6.9 yesterday.
Tried again today with the fix above (I tried yesterday too, btw.) and it worked like a charm. Thanks for that.

Long story short: Running `python.exe -m cddagl` manually in Powershell taught me again, that it's a bad idea to try to run commands made for the commandline in PS.
Just FYI in case someone else tries to repro the error message in Powershell instead of the commandline and gets a mild headache.
